middleware/index.ts

```diff
@@ -282,6 +284,58 @@ function errorMessage(error: Error): any {
         .build();
 }
 
+function isInitialAsyncRequest(payload: any, headers: any): boolean {
+    return payload.type 
+        && payload.type.startsWith('com.salesforce.function.async')
+        && !headers[ASYNC_HEADER];
+}
+
+// Invoke given function again to fulfill async request; response is not handled
+async function invokeAsyncFn(logger: Logger, payload: any, headers: any): Promise<void> {
+    let hostUrl = headers['Host'] || payload?.data?.sfContext?.resource;
```

Comment:
Is there a reason we prefer the host over the context resource?

Reply:
Good question.  Making use of a standard header.  I believe Core sends.  `:invoke` needs updating.  If a client doesn't provide, we'll fallback to `resource`, although `sfContext` is internal.  Seems like there should be a place to store as top-level CloudEvent attr.